dvl-a50/mavlink2resthelper.py

```diff
@@ -1,24 +1,26 @@
-from blueoshelper import request, post
-import requests
 import json
 import time
 from math import radians
 from typing import Any, Optional
+
+import requests
 from loguru import logger
 
+from blueoshelper import post, request
+
 MAVLINK2REST_URL = "http://127.0.0.1/mavlink2rest"
 GPS_GLOBAL_ORIGIN_ID = 49
 
 # holds the last status so we dont flood it
 last_status = ""
 
-
+# pylint: disable=too-many-instance-attributes
```

Comment:
By the documentation I see that the limit is 7 but here I only see 4 attributes for the class, is this correct ?